libsel4vmmplatsupport/src/arch/arm/guest_vcpu_util.c

```diff
@@ -87,6 +91,7 @@ int fdt_generate_plat_vcpu_node(vm_t *vm, void *fdt)
     int root_offset = fdt_path_offset(fdt, "/");
     int cpu_node = fdt_add_subnode(fdt, root_offset, "cpus");
     if (cpu_node < 0) {
+        ZF_LOGE("Failed to create cpu node");
         return cpu_node;
     }
     FDT_OP(fdt_appendprop_u32(fdt, cpu_node, "#address-cells", 0x1));
```

Comment:
I lack some background about the code, so I wonder what happens now if `cpu_node` is '0`? Doesn't this expect to get some handle here? Or can this handle `0` then also?

Reply:
What I wonder is, does it make sense to pass `0` there now of should we also check for `if (0 == cpu_node) {` now also and don't try to do anything at all then - do we don't run into error that are expected anyway, which could be confusing.

Reply:
If `fdt_add_subnode`'s return value is >= 0, then it is supposed to be interpreted as an offset to the newly created node.